src/Physics/HadronTransport/2025/HAIntranuke2025.cxx

```diff
@@ -0,0 +1,1631 @@
+//____________________________________________________________________________
+/*
+ Copyright (c) 2003-2024, The GENIE Collaboration
+ For the full text of the license visit http://copyright.genie-mc.org
+ 
+
+ Author: Steve Dytman <dytman+@pitt.edu>, Pittsburgh Univ.
```

Comment:
Is this author list accurate?  The  date at the end suggests that this is a from a copy of another file.